mdata/cache/accnt/lru.go

```diff
@@ -24,6 +24,10 @@ func (l *LRU) touch(key interface{}) {
 	}
 }
 
+func (l *LRU) del(key interface{}) {
+	delete(l.items, key)
```

Comment:
if it's ok for the delete not to affect the `list`, then a comment should explain why

Reply:
you're right, it actually should also delete from `.list`

Reply:
https://github.com/grafana/metrictank/pull/555/commits/3439fb0ab49ba2c2f1e89c37c6d1ef6def3ee841